.eslintrc

```diff
@@ -15,6 +15,7 @@
     "react"
   ],
   "rules": {
+    "no-console": "off",
```

Comment:
I often use console logs for development. I can put it back if you prefer to keep it disabled.

Reply:
I do too but I prefer to keep it disabled to avoid accidentally merging the many `console.log("WTF WHY ISN'T THIS WORKING');` statements I often put into my code 😄

Reply:
+1 to keeping eslint checking for console statements, I also tend to leave them in my code and would like eslint to check for this